test/e2e/parallel/banned_user_test.go

```diff
@@ -0,0 +1,78 @@
+package parallel
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	testspace "github.com/codeready-toolchain/toolchain-common/pkg/test/space"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	testsupportspace "github.com/codeready-toolchain/toolchain-e2e/testsupport/space"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBannedUser(t *testing.T) {
```

Comment:
one more thing - since it's specific to the proxy, should we move it to the `proxy_test.go` ?

Reply:
yeah, renaming sounds good me too! I don't think there is any plan to split it for now. but sounds a good idea. That said we could just rename the test and file so it's clear that is for proxy only and not banned users in general which is covered in the `user_management_test.go`

Reply:
done in d3ea52a10ddb8199ed1fbd799b15bc0cf8c13187